manifest.json

```diff
@@ -4,15 +4,18 @@
   },
   "auth": "api_key",
   "datatypes": {
+    "issues": {
+      "template": "templates/projects.json"
+    },
     "projects": {
       "template": "templates/projects.json"
     },
     "users": {
-      "template": "templates/projects.json"
+      "template": "templates/users.json"
     },
-    "issues": {
-      "template": "templates/projects.json"
+    "users-youtrack": {
```

Comment:
Don't put dashes in datatype names. This should be usersyoutrack. But also, if the set of users returned by usersyoutrack-list is the same as the set returned by users-list AND the set of data contained in usersyoutrack-list exists in the set of data in users-list, then as mentioned in the standup you don't need the usersyoutrack datatype at all.

Reply:
I've removed the dashes now.